.changeset/little-pillows-tell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@pantheon-systems/wordpress-kit": patch
```

Comment:
This is actually a case where a single changeset for all of these might make more sense xD. No need to change it right now, but whenever the change is the same across multiple packages, I think in that case a single changeset is fine. We should probably come up with some guidelines around the changesets at some point.

Reply:
That makes sense. I think eventually having some official changeset guidelines would be helpful and worth doing😆